data/net/minecraft/client/renderer/entity/LightningBoltRenderer.mapping

```diff
@@ -3,8 +3,8 @@ CLASS net/minecraft/client/renderer/entity/LightningBoltRenderer
 		COMMENT Returns the location of an entity's texture.
 		ARG 1 entity
 	METHOD quad (Lorg/joml/Matrix4f;Lcom/mojang/blaze3d/vertex/VertexConsumer;FFIFFFFFFFZZZZ)V
-		ARG 0 matrix
-		ARG 1 consumer
+		ARG 0 pose
+		ARG 1 buffer
```

Comment:
RE: Why the change from `consumer` to `buffer` for type `VertexConsumer`?

Reply:
`buffer` is more correct. The `VertexConsumer` is always sourced from `MultiBufferSource.getBuffer()` and represents a buffer.